src/board/properties/property.js

```diff
@@ -27,6 +23,34 @@ Property.prototype.performLandingAction = function(game) {
   Property.super_.prototype.performLandingAction.call(this, game);
 };
 
+Property.prototype.canBuy = function(game) {
+  var player = game.getCurrentPlayer();
+  if (this.owner === null) {
+    if (player.money > this.cost) {
+      return true;
+    }
+  } 
+  return false;
+};
+
+Property.prototype.oweRent = function(game) {
+  var player = game.getCurrentPlayer();
+  if (this.owner !== null){
+    if (this.owner !== player) {
+        return true;
+    }
+  }
+  return false;
+};
+
+Property.prototype.own = function(game) {
```

Comment:
I think this might be a better method on the player object? Or it should be reworded. I think `player.own(property)` reads better